src/tools/OptionsSchema.json

```diff
@@ -463,6 +463,11 @@
           "default": {
             "*": { "enabled": true }
           }
+        },
+        "useSnapshotDebugger": {
+          "type": "boolean",
+          "description": "If true then Snapshot Debugger will be used rather than live debugger.",
```

Comment:
Do we want this to be a user-exposed option? If not, there are better places to put this. If so, we need a better description?

Reply:
User's shouldn't have to mess with this manually. Let's discuss where else to put it.